Python/pywarpx/picmi.py

```diff
@@ -4249,6 +4252,46 @@ def _handle_particle_histogram(self, **kw):
 
         return kw
 
+    def _handle_particle_histogram2d(self, **kw):
+        self.bin_number_abs = kw.pop("bin_number_abs")
+        self.bin_number_ord = kw.pop("bin_number_ord")
+        self.bin_min_abs = kw.pop("bin_min_abs")
+        self.bin_max_abs = kw.pop("bin_max_abs")
+        self.bin_min_ord = kw.pop("bin_min_ord")
+        self.bin_max_ord = kw.pop("bin_max_ord")
+        histogram_function_abs = kw.pop("histogram_function_abs")
+        histogram_function_ord = kw.pop("histogram_function_ord")
+        self.__setattr__(
+            "histogram_function_abs(t,x,y,z,ux,uy,uz,w)", histogram_function_abs
+        )
+        self.__setattr__(
+            "histogram_function_ord(t,x,y,z,ux,uy,uz,w)", histogram_function_ord
+        )
+
+        filter_function = kw.pop("filter_function", None)
+        value_function = kw.pop(
+            "filter_function", "1."
```

Comment:
This PR looks good overall - thanks! Note that this should default to None since the default values are already handled in the C++.

Also, don't forget to add the documentation for the new input arguments.